src/turbomind/models/llama/LlamaBatch.cc

```diff
@@ -1131,6 +1134,7 @@ void LlamaBatch<T>::InitializeSampling(const GenerationState& g)
         }
     }
     outputs_ = std::move(outputs);
+    sync_check_cuda_error();
```

Comment:
Why should we `cudaDeviceSynchronize` here?

Reply:
It will sync only when TM_DEBUG_LEVEL=DEBUG, I add here to better calculate the time